sycl/test-e2e/Sampler/basic-rw.cpp

```diff
@@ -15,7 +15,7 @@
     ONEAPI_DEVICE_SELECTOR=level_zero:gpu ./binx.bin
     ONEAPI_DEVICE_SELECTOR=opencl:cpu ./binx.bin
 
-    ONEAPI_DEVICE_SELECTOR=opecl:acc ../binx.bin    <--  does not support image
+    ONEAPI_DEVICE_SELECTOR=opencl:fpga ../binx.bin    <--  does not support image
```

Comment:
accelerator is what our OpenCL plugin reports and also what we document in the oneapi-construction-kit i how to use with dpc++ - are you saying that "fpga" will map onto this?

Reply:
As per the ONEAPI_DEVICE_SELECTOR [documentation](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#oneapi_device_selector), the device type can only be cpu, gpu, or fpga (or any combination of those). Currently, 'acc' is also accepted by ONEAPI_DEVICE_SELECTOR as a valid device type, which is incorrect.
Therefore, I'm changing test cases and documentation, that uses ONEAPI_DEVICE_SELECTOR with 'acc' device type to instead use 'fpga'. Functionality wise, **'acc' and 'fpga' device types are equivalent** so this change should not change the functionality of the test cases.

Reply:
I don't know about the test cases, I just wanted to be sure we can put fpga when the opencl device says accelerator.